ui/page/main_page.go

```diff
@@ -192,14 +202,12 @@ func (mp *MainPage) initNavItems() {
 // Part of the load.Page interface.
 func (mp *MainPage) OnNavigatedTo() {
 	// register for notifications, unregister when the page disappears
```

Comment:
This comment seems misplaced, the next line doesn't register for notifications.

Reply:
I did not add this comment. I'll remove it since it isn't relevant to that section of the code.